### DISCUSSION TITLE
Skill Porting Standards

### DISCUSSION THREAD

**Discussion by jmillerv** [jmillerv/mycroft2-skills#1] [General]
##  Welcome!
 I set up this discussion board to try and build out some standards for converting skills to Dinkum. Hopefully we can use this to build out the standards for the repo. 

I've never maintained a project and was wondering if anyone had suggestions on a good contribution.md to pull from?

**Comment by JarbasAl** [comment]
re contributing.md , have a look here https://github.com/OpenVoiceOS/community-docs/pull/11

**Reply by jmillerv** [reply]
Cool, I'll make a branch and add this to it with changes appropriate for the mycroft repo.
